mathesar/imports/csv.py

```diff
@@ -22,8 +22,8 @@ def create_db_table_from_csv(name, schema, csv_reader, engine):
 
 def create_table_from_csv(name, schema, csv_file, engine=engine):
     csv_reader = get_csv_reader(csv_file)
-    table = create_db_table_from_csv(name, schema, csv_reader, engine)
-    table, _ = Table.objects.get_or_create(
-        name=table.name, schema=table.schema
-    )
+    db_table = create_db_table_from_csv(name, schema, csv_reader, engine)
+    database = engine.url.database
```

Comment:
I think it would be better (if possible) to instead use the key associated with `settings.DATABASES` for the DB, so all the relevant connection info can be retrieved using the data in the model (and `settings.py`).  Or, we could store the whole connection string, but that would have sensitive info (the password).

Reply:
That makes sense, I'll update it.